docs/src/transforms.md

```diff
@@ -293,6 +293,11 @@ If SCRAM authentication against the first kafka broker fails, shotover will term
     # When a timeout occurs the connection to the client is immediately closed.
     # read_timeout: 60
 
+    # Delay in milliseconds before the local shotover node checks its next peer's node state.
+    # The local shotover node will try to establish a TCP connection with the next peer in the list.
+    # If the connection cannot be established within connect_timeout_ms, then the peer is considered down.
```

Comment:
```
Delay in milliseconds before the local shotover node checks its next peer's node state.
```
Reading this sentence before the other sentences is confusing because we dont have enough context yet. I think it should make more sense if we swap up the order.

```
Shotover will regularly open a TCP connection to each of its peers to check if they are up or down.
If shotover detects that a peer is down shotover will exclude the down peer from its metadata reports to the client.
Each peer is checked in a round robin fashion and this `check_shotover_peers_delay_ms` field defines the milliseconds delay taken before moving onto the next peer to check.
If the connection cannot be established within connect_timeout_ms, then the peer is considered down.
```

Reply:
updated

Reply:
Was the first line: `Shotover will regularly open a TCP connection to each of its peers to check if they are up or down.` intentionally excluded?

Reply:
my bad, it was missed